frontend/src/containers/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import { houseIssuesFetchData} from './actions';
+import {HouseList} from '../../components/House';
+
+class Dashboard extends Component {
+  componentDidMount = () => {
+    this.props.fetchData( '/api/v1/house-issues');
```

Comment:
We need to agree endpoint syntax

Reply:
What do you mean ?

Reply:
/api/v1/house/id/issues for example ... not house-issues